client/post-editor/editor-revisions-list/view-buttons.jsx

```diff
@@ -0,0 +1,66 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SegmentedControl from 'components/segmented-control';
+import ControlItem from 'components/segmented-control/item';
+import {
+	splitPostRevisionsDiffView,
+	unifyPostRevisionsDiffView,
+} from 'state/posts/revisions/actions';
+import { getPostRevisionsDiffView } from 'state/selectors';
+
+const EditorRevisionsListViewButtons = ( { translate, diffView, viewSplit, viewUnified } ) => {
+	return (
+		<SegmentedControl compact className="editor-revisions-list__view-buttons">
```

Comment:
Does lint complain here? I think I would expect `editor-revisions-list-view-buttons` based on the function name here.

Reply:
The class name format is not based on the component name, but on the folder.
In this case we're in `/editor-revisions-list/view-buttons.jsx`, so the class **must** begin with `editor-revisions-list__`.
Then, we can add pretty much anything after that, and in this case `view-buttons` is the best choice.

See for example what happens in the good ol' comments section:
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/comments/comment/comment-edit.jsx#L106

`/comment/comment-edit.jsx` ➡️ `comment__edit`

Notice that it's `__edit` and not `__comment-edit`, because eslint doesn't really care about what happens after the folder name, and in that case it made no sense to have a repetition like `comment__comment-edit`.

Reply:
Thanks Jacopo, good to know.